pkg/ucp/aws/ucpcredentialprovider.go

```diff
@@ -43,7 +43,7 @@ const (
 	// CredentialKind is AccessKey
 	CredentialKindAccessKey = "AccessKey"
 	// Token file path for IRSA
-	tokenFilePath = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
+	TokenFilePath = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
```

Comment:
Did we start using this outside of this folder? Is that the reason why you capitalized it?

Reply:
yes its is the same path we use for token file for ecr auth, so made it global